features/scripts/setup_gradle_wrapper.sh

```diff
@@ -5,40 +5,40 @@ export UPDATING_GRADLEW=true
 
 pushd "$APP_FIXTURE_DIR"
 echo "Updating gradle wrapper for: $APP_FIXTURE_DIR"
-./gradlew wrapper --gradle-version "$GRADLE_WRAPPER_VERSION"
+sed -i.bak -e "s/^distributionUrl.*$/distributionUrl=https\\\\:\/\/services.gradle.org\/distributions\/gradle-$GRADLE_WRAPPER_VERSION-bin.zip/" gradle/wrapper/gradle-wrapper.properties
```

Comment:
Seems worthwhile to put this in a tiny function at the top of the file, since its the same somewhat-complex bits repeated a number of times.

Reply:
:man_cook: :ok_hand: